### PR TITLE
Package Pulse for deploy

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR introduces some changes needed to get deployment of pulse going.

It introduces some refactoring to make the packages behave nicely when installed in non-editable mode, and adds a Dockerfile for `pulse`.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,5 @@
 recursive-include pulse/templates *
 recursive-include pulse/static *
+recursive-include data/ineligible *
```
Comment (buckley-w-david) [inline MANIFEST.in]: This can likely be 🔥 at some later point, am just including it as the current system breaks without it.

File: data/env.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 import yaml
 
 DATA_DIR = os.path.dirname(__file__)
 
 # App-level metadata.
-META = yaml.safe_load(open(os.path.join(DATA_DIR, "../meta.yml")))
+_resource_package = __name__
+_resource_path = 'data_meta.yml'
+_meta_content = pkg_resources.resource_stream(_resource_package, _resource_path)
+META = yaml.safe_load(_meta_content)
```
Comment (buckley-w-david) [inline data/env.py]: This is how how to read in a file that has been packaged with the code

File: data/processing.py [hunk 3]
```diff
@@ -163,27 +164,30 @@ def run(date):
   # Overwrites `domains` and `subdomains` in-place.
   process_domains(domains, agencies, subdomains, parent_scan_data, subdomain_scan_data)
 
+  _app = create_app('production')
```
Comment (buckley-w-david) [inline data/processing.py]: This needs to be changed.

File: MANIFEST.in (hunk 1 above)
Comment (evadb) [reply MANIFEST.in]: yeah, agreed with the 🔥 since it's specific to 18Fs pulse, but in another PR

File: data/env.py (hunk 2 above)
Comment (evadb) [reply data/env.py]: I wondered about this! But I really like the format, it's nice and clear

**Pull request merged**